vultr/config.go

```diff
@@ -19,7 +20,7 @@ type Client struct {
 
 // Client configures and returns a fully initialized Vultr Client.
 func (c *Config) Client() (interface{}, error) {
-	client := Client{lib.NewClient(c.APIKey, &lib.Options{})}
+	client := Client{lib.NewClient(c.APIKey, &lib.Options{RateLimitation: 2 * time.Second})}
```

Comment:
Why did you choose `2 * time.Second`? If the vultr'S default API rate limit is two requests per second, shouldn't this be `0.5 * time.Second`?